schema/cosa/schema_doc.go

```diff
@@ -474,6 +475,12 @@ var generatedSchemaJSON = `{
          "type":"object",
          "title":"Qemu",
          "$ref": "#/definitions/artifact"
+       },
+       "secex-qemu": {
+         "$id":"#/properties/images/properties/se",
+         "type":"object",
+         "title":"SecureExecution Qemu",
```

Comment:
Minor: is the correct styling "SecureExecution" or "Secure Execution"? The official IBM docs seem to imply the latter.